### PR TITLE
DROOLS-3799: Check and fix i18n

### PR DESCRIPTION
**Pull request by yesamer**
@dupliaka @danielezonca can you please review and test it?
No changes on functionality, I just removed all hard coded label and replaced with i18n labels.

https://issues.redhat.com/browse/DROOLS-3799

### CODE REVIEW

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/SettingsViewImpl.java [hunk 1]
```diff
@@ -97,12 +98,18 @@
...
 
     @Override
     public void init(Presenter presenter) {
         this.presenter = presenter;
+        nameLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.name());
+        typeLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.type());
+        statelessLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.statelessLabel());
+        dmnFileLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.dmnModelLabel());
+        dmnNamespaceLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.dmnNamespaceLabel());
+        dmnNameLabel.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.dmnNameLabel());
```
Comment (dupliaka) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/SettingsViewImpl.java:112]: How about add this check too?
chooseButton.setTitle(ScenarioSimulationEditorConstants.INSTANCE.chooseFile());

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/collectioneditor/ElementViewImpl.java [hunk 2]
```diff
@@ -42,27 +43,39 @@
...
+    protected ButtonElement cancelChangeButton = Document.get().createPushButtonElement();
+
+    @DataField("cancelButtonSpanText")
+    protected SpanElement cancelButtonSpanText = Document.get().createSpanElement();
 
     @Override
     public void init(Presenter presenter) {
         this.presenter = presenter;
+        itemSeparatorText.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.item());
+        saveChangeButtonSpanText.setInnerText(ScenarioSimulationEditorConstants.INSTANCE.saveButton());
```
Comment (dupliaka) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/collectioneditor/ElementViewImpl.java:77]: How about add some test on init in ElementViewImplTest?

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties [hunk 3]
```diff
@@ -138,12 +138,19 @@ removeCollectionQuestion=Do you want to continue and remove the collection?
 removeCollectionWarningText=This action cannot be undone.
 #
 settings=Settings
+name=Name
+type=Type
+statelessLabel=Stateless Session
+dmnModelLabel=DMN model
+dmnNamespaceLabel=DMN namespace
```
Comment (danielezonca) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties:145]: Not sure if it makes sense to translate these words to be honest

Comment (yesamer) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties]: @danielezonca Yes, we can consider them as technical terms, so fine for me to don't translate them. Done.

**Comment by yesamer** [comment]
@danielezonca Can you please merge it?

**Pull request merged**